core/src/main/java/io/lionweb/lioncore/java/language/NamespacedEntity.java

```diff
@@ -6,8 +6,7 @@
  * <p>A Concept Invoice, contained in a Language com.foo.Accounting. Therefore, Invoice will have
  * the qualifiedName com.foo.Accounting.Invoice.
  */
-public interface NamespacedEntity {
-  // TODO add ID, once details are clearer
+public interface NamespacedEntity extends INamed {
```

Comment:
This is not part of "official" M3, but as discussed an implementation can have other interfaces. Just to make sure this is a conscious decision.